packages/rocketchat-livechat/server/methods/getAnalyticsChartData.js

```diff
@@ -0,0 +1,88 @@
+import moment from 'moment';
+
+Meteor.methods({
+	'livechat:getAnalyticsChartData'(options) {
+		if (!Meteor.userId() || !RocketChat.authz.hasPermission(Meteor.userId(), 'view-livechat-manager')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', {
+				method: 'livechat:getAnalyticsChartData'
+			});
+		}
+
+		const from = moment(new Date(options.daterange.from));
+		const to = moment(new Date(options.daterange.to));
+
+		const data = {
+			chartLabel: options.chartOptions.name,
+			dataLabels: [],
+			dataPoints: []
+		};
+
+		for (let m = moment(from); m.diff(to, 'days') <= 0; m.add(1, 'days')) {
+			data.dataLabels.push(m.format('M/D/YY'));
+
+			const date = {
+				gte: m,
+				lt: moment(m).add(1, 'days')
+			};
+
+			switch (options.chartOptions.name) {
+				case 'Total_conversations':
+					//
+					data.dataPoints.push(RocketChat.models.Rooms.getTotalConversationsBetweenDate('l', date));
+					break;
+				case 'First_response_time':
```

Comment:
Please, think about moving the switch statement to methods(props) inside `RocketChat.Livechat.Analytics`, like 
RocketChat.Livechat.Analytics[options.chartOptions.name](options)..
Centralizing analytics stuff it the `Analytics` lib, it will help in tracking issues in the future.